### ISSUE TITLE
bug: accounts.contact does not exist

### ISSUE THREAD

**Issue by trev-dev** [nix-community/home-manager#4344]
### Are you following the right branch?

- [X] My Nixpkgs and Home Manager versions are in sync

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue description

I am attempting to set up vdirsyncer via `accounts.contact.accounts.<name>` and I am getting the following error:

```
error: The option `accounts.contact' does not exist. Definition values:
       - In `/home/trev/.config/home-manager/email.nix':
           {
             accounts = {
               fastmail = {
                 local = {
                   fileExt = ".vcf";
           ...
(use '--show-trace' to show detailed location information)
```

Is this implementation maybe not ready to go yet?

### Maintainer CC

CC: @teto 

### System information

```markdown
- system: `"x86_64-linux"`
 - host os: `Linux 6.1.43, NixOS, 23.05 (Stoat), 23.05.2573.61676e4dcfee`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - channels(root): `"nixos-23.05"`
 - channels(trev): `"home-manager-23.05.tar.gz, unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by antonmosich** [comment]
The contact/calendar features are rather recent. They have been added in June and are not included in the 23.05 release of home-manager. I believe you are using that though, so it's not yet available.

**Comment by trev-dev** [comment]
@antonmosich Thanks
